File: okcoin.js
var growl = require('growl')
/*growl('you have an email',{sticky:true})*/
var request = require('request')
var url = 'https://www.okcoin.com/api/ticker.do?symbol=ltc_cny'
var price = 0
function ok(request,url,price){
request({uri:url},function(err,response,body){
		var result = JSON.parse(response.body)
		var newprice = result.ticker.sell
		if(newprice > price){
			growl(newprice,{title:'Rising',image:'/Users/latpaw/Downloads/favicon.png'})
		}else if(newprice < price){
			growl(newprice,{title:'Downing',image:'~/Downloads/favicon.png'})
		}else{growl(price,{title:'Stable',image:'~/Downloads/favicon.png'})}
	})
}
setInterval(function(){ok(request,url,price)},5000)

File: okcoin.rb
require "nokogiri"
require "open-uri"
require "json"
price = 0
while true do
  doc = Nokogiri::HTML(open("https://www.okcoin.com/api/ticker.do?symbol=ltc_cny"))
  result = JSON.parse doc.content
  newprice = result["ticker"]["sell"]
  if newprice.to_f > price 
    system "growlnotify -m \"Rising,#{newprice}\""
    price = newprice.to_f
  elsif newprice.to_f < price
    system "growlnotify -m \"downing,#{newprice}\""
    price = newprice.to_f
  else
    system "growlnotify -m \"no change,#{price}\""
  end
  
    sleep 20
end
